torch/_tensor_docs.py

```diff
@@ -664,6 +664,20 @@ def add_docstr_all(method, docstr):
 In-place version of :meth:`~Tensor.bitwise_not`
 """)
 
+add_docstr_all('bitwise_invert',
+               r"""
+bitwise_invert() -> Tensor
+
+See :func:`torch.bitwise_invert`
```

Comment:
Should we also mention that this is an alias for `bitwise_not`.

Reply:
For whatever reason we typically have the method point to the function variant and then the function variant says it's an alias.